include/swift/AST/IRGenOptions.h

```diff
@@ -65,6 +65,8 @@ enum class IRGenDebugInfoFormat : unsigned {
   CodeView
 };
 
+enum class IRGenLLVMLTOKind : unsigned { None, Thin, Full };
```

Comment:
Too bad clang-format does this. I may try to fix this at some point. Don't worry about it though. Not on you.

Reply:
FYI, adding trailing comma after `Full` results,
```cpp
enum class IRGenLLVMLTOKind : unsigned {                                        
  None,                                                                         
  Thin,                                                                         
  Full,                                                                         
};
```